src/bindings/js/node/src/compiled_model.cpp

```diff
@@ -46,3 +73,37 @@ Napi::Value CompiledModelWrap::get_outputs(const Napi::CallbackInfo& info) {
 
     return js_outputs;
 }
+
+Napi::Value CompiledModelWrap::get_input(const Napi::CallbackInfo& info) {
+    if (info.Length() == 0) {
+        try {
+            return Output::Wrap(info.Env(), _compiled_model.input());
+        } catch (std::exception& e) {
+            reportError(info.Env(), e.what());
+            return Napi::Value();
+        }
+    } else if (info.Length() != 1) {
+        reportError(info.Env(), "Invalid number of arguments -> " + std::to_string(info.Length()));
+        return Napi::Value();
+    } else if (info[0].IsString()) {
+        auto tensor_name = info[0].ToString();
+        return Output::Wrap(info.Env(), _compiled_model.input(tensor_name));
+    } else if (info[0].IsNumber()) {
+        auto idx = info[0].As<Napi::Number>().Int32Value();
+        return Output::Wrap(info.Env(), _compiled_model.input(idx));
```

Comment:
If passed not existed index, error appears.
Do we need to catch this error or original looks good on js side?

Reply:
Original error from C++ says: 

> Check 'i < _impl->outputs().size()' failed at src/inference/src/compiled_model.cpp:86:
Cannot get output for index: 6 outputs size is 1

I think it is clear where the error comes from, but when we add tests on js side (e.g using JEST) we will have to report this error using Napi::Error which is a representation of the JavaScript Error object.